### PR TITLE
Website update

### PR DESCRIPTION
**Pull request by pwang14**
- Updated information and links
- New page for bookings
- Don't worry about the JavaScript stuff, it's all commented out

### CODE REVIEW

**Comment by praihan** [comment]
Hi there @gustavtrolle! Thank you for the contribution. I'm glad that this website is still going!

It appears that your PR includes already built files.

Notice [public/assets/](https://github.com/gustavtrolle/cblib.site/tree/22f61ada2ea439e766bf350aad8dce1bad5a256e/public/assets) (minified) and [public/](https://github.com/gustavtrolle/cblib.site/tree/22f61ada2ea439e766bf350aad8dce1bad5a256e/public/) (unminified) are both included.

Only the unminified code should be checked in to Git. I'm unsure how those files got there for you. Following the [README.md](https://github.com/prshreshtha/cblib.site/blob/f897e37b7e94a0ecff9e3ec8e37ccd85e3dfbea8/README.md), all those should have ended up in `deploy/`.

Could you clean up the files? You claim that your changes include:
> * Updated information and links
> * New page for bookings

This should only touch a few files, not 171.

Cheers,
Pranjal

**Comment by praihan** [comment]
As an aside, I can add you as a collaborator if you will be making changes to the website from here on. I see your changes are already live at http://colonelby.com/teachers/library/

**Comment by pwang14** [comment]
Sorry about that! I was transferring the website files to my new laptop and I guess something must've gone wrong. I have removed the additional files.

Also, another student will be in charge of the website, but I don't have his GitHub account at the moment. You can add him as a collaborator later.

Thanks,
Patrick

File: public/assets/js/vendor/k.js [hunk 1]
```diff
@@ -1,105 +1,149 @@
...
+        code2: function (link) {
+            window.location = link
+        },
+        code3: function (link) {
+            window.location = link
+        },
+        code4: function (link) {
+            window.location = link
+        },
+        /*codeD: function (link) {
```
Comment (praihan) [inline public/assets/js/vendor/k.js:84]: Can you explain what these are? There's no reason to leave commented out code if it's in Git. You can always recover them from the commit history if needed.

**Review by praihan — CHANGES_REQUESTED** [review]
If you're adding more konami codes, please let me know what they are!

**Comment by praihan** [comment]
I see that many of the files changed modes from `10644` to `10755`. This is from adding executable permission bits but these files should not be executable. Can you set them back?

**Comment by praihan** [comment]
> Also, another student will be in charge of the website, but I don't have his GitHub account at the moment. You can add him as a collaborator later.

Sure thing. Point him to me if you get the chance.

**Pull request closed without merging**